app/views/tasks/_form.html.slim

```diff
@@ -100,20 +100,20 @@
             = f.nested_fields_for :tests do |test|
               .show-table.exercise-show
                 - if test.object.new_record?
-                  - display_container = 'display: block'
+                  - display_container_class = 'd-block'
                   .table-header
                     = test.remove_nested_fields_link :text, class: 'btn btn-main btn-sm float-end' do
                       i.fa-solid.fa-xmark.text-danger
                       =< t('.button.remove_test')
                 - else
-                  - display_container = 'display: none'
+                  - display_container_class = 'd-none'
                   .table-header.closed.toggle-next
                     => test.object.title
                     span.fa-solid.fa-caret-down
                     = test.remove_nested_fields_link :text, class: 'btn btn-main btn-sm float-end' do
                       i.fa-solid.fa-xmark.text-danger
                       =< t('.button.remove_test')
-                .test-container.ms-2.mt-4 style="#{display_container}"
+                .test-container.ms-2.mt-4 class="#{display_container_class}"
```

Comment:
shouldn't this be enough?

```suggestion
                .test-container.ms-2.mt-4 class=display_container_class
```

Reply:
Thanks. I have corrected it.

Reply:
Ups, the suggested change broke the slim/test.

```
app/views/tasks/_form.html.slim:103 [W] RuboCop: Lint/UselessAssignment: Useless assignment to variable - `display_container_class`.
app/views/tasks/_form.html.slim:109 [W] RuboCop: Lint/UselessAssignment: Useless assignment to variable - `display_container_class`.
```

Should we leave it as it is, or there is some workaround?

Reply:
we have the same problem in `app/views/tasks/_nested_file_form.html.slim` where we disabled the cop for these cases. Maybe `"#{display_container_class}"` would be better after all or you could also disable the cop. Up to you, but do it consistently :)

Reply:
Since we already suppress similar lint warnings, I also use the same approach here.